easybuild/easyconfigs/b/binutils/binutils-2.31.1-gold-ignore-discarded-note-relocts.patch

```diff
@@ -0,0 +1,11 @@
+diff -ru binutils-2.31.1.orig/gold/target-reloc.h binutils-2.31.1/gold/target-reloc.h
```

Comment:
@terjekv Please add a short patch description on top of this file (anything above the `---` line is ignored by `patch`), as well as a reference to where this patch was obtained from.

Keep in mind that this implies updating the SHA256 checksums as well

Reply:
Comments added. Tested by rebuilding all affected versions of binutils, and then rebuilding LLVM and Bazel that caused the errors beforehand.